test/packages/good_lookup_index/elasticsearch/transform/foo/manifest.yml

```diff
@@ -0,0 +1,5 @@
+start: true
+destination_index_template:
+  settings:
+    index:
+      mode: "lookup"
```

Comment:
Is this feature only intended for transforms? If it is, we could maybe add this only to transforms. If it isn't could you add the setting to the data stream manifest?